### PR TITLE
refactor : make product listing page responsive

### PR DESCRIPTION
**Pull request by viru2001**
## make product listing page responsive

responsive products page

### CODE REVIEW

**Review by ckahub — COMMENTED** [review]
Looks good to merge

**Review by aurobindoGupta — APPROVED** [review]
nice work!!

File: styles/products.css [hunk 1]
```diff
@@ -47,3 +51,23 @@ footer {
...
+  .wrapper {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      "header header"
+      "hamburger-menu hamburger-menu"
+      "filters filters"
+      "products products"
+      "footer footer";
+  }
+  #filters{
```
Comment (SHIKHASINGH1506) [inline styles/products.css:65]: is there any specific reason for id selector here? id selector is only used for very unique elements as per my knowledge.

**Review by SHIKHASINGH1506 — APPROVED** [review]

File: styles/products.css (hunk 1 above)
Comment (viru2001) [reply styles/products.css:65]: filters section is unique on products page that's why I used id

**Pull request merged**